src/components/container/MostRecentNotification/MostRecentNotification.tsx

```diff
@@ -1,27 +1,51 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState } from 'react'
 import "./MostRecentNotification.css"
-import MyDataHelps, { NotificationQueryParameters, NotificationType, Notification } from "@careevolution/mydatahelps-js"
-import { Action, SingleNotification } from '../../presentational'
+import MyDataHelps, { NotificationQueryParameters, NotificationType, Notification, QueryParameters } from "@careevolution/mydatahelps-js"
+import { Action, LoadingIndicator, SingleNotification } from '../../presentational'
 import { add } from 'date-fns'
-import { previewNotification } from './MostRecentNotification.previewdata'
+import { previewNotifications } from './MostRecentNotification.previewdata'
 import language from '../../../helpers/language'
+import { useInitializeView } from '../../../helpers'
 
 export interface MostRecentNotificationProps {
 	notificationType?: NotificationType,
+	notificationIdentifierRegex?: RegExp;
 	onViewMore?: Function,
 	hideAfterHours?: number,
 	previewState?: MostRecentNotificationPreviewState
 	innerRef?: React.Ref<HTMLDivElement>
 }
 
-export type MostRecentNotificationPreviewState = "Default";
+export type MostRecentNotificationPreviewState = 'loading' | 'loaded with data' | 'loaded with no data';
 
 export default function (props: MostRecentNotificationProps) {
 	const [notification, setNotification] = useState<Notification | null>(null);
+	const [loading, setLoading] = useState<boolean>(true);
 
-	function initialize() {
-		if (props.previewState == "Default") {
-			setNotification(previewNotification);
+	async function initialize() {
+		setLoading(true);
+		if (props.previewState === 'loading') {
+			setNotification(null);
```

Comment:
Not 100% positive, but I think this is actually breaking this story.  At the very least, it appears that not having `setLoading(true)` in here breaks it?  Right now the story is showing nothing when it was previously showing a spinner.

Reply:
Sorry about being sloppy here. I'll fix all this up